frontend/src/app/models/create-coworking-method/create-coworking-method.component.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright DB Netz AG and the capella-collab-manager contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Component, OnInit } from '@angular/core';
+import {
+  AbstractControl,
+  AsyncValidatorFn,
+  FormControl,
+  FormGroup,
+  ValidationErrors,
+  Validators,
+} from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { merge, Observable } from 'rxjs';
+import { GitModel, ModelService } from 'src/app/services/model/model.service';
+import { ProjectService } from 'src/app/projects/service/project.service';
+import {
+  Credentials,
+  GitService,
+  Instance,
+} from 'src/app/services/git/git.service';
+import { Source, SourceService } from 'src/app/services/source/source.service';
+
```

Comment:
`GitModel`  and `AsyncValidatorFn` can be deleted.

Reply:
I removed these useless imports and some others.